src/main/java/hu/wurfel/refference/school/division/ClassController.java

```diff
@@ -25,12 +25,12 @@ public ResponseEntity<List<Class>> create(@RequestBody ClassRequestForSearch cla
         return ResponseEntity.ok(classService.save(Integer.parseInt(classRequestForSearch.getId()), Short.parseShort(classRequestForSearch.getGrade()), classRequestForSearch.getSign().strip().charAt(0), Year.parse(classRequestForSearch.getYear()), Long.parseLong(classRequestForSearch.getTeacherId())));
     }
 
-    @DeleteMapping("/delete/{id}")
+    @DeleteMapping("/{id}")
     public void delete(@PathVariable String id) {
         classService.delete(classService.getByClassId(Integer.parseInt(id)));
     }
 
-    @PutMapping("/modify")
+    @PutMapping("/")
```

Comment:
```suggestion
    @PutMapping
```
Ha nincs semmi a / után, akkor nem kell. Frontenden össze tud zavarni dolgokat.

Reply:
kivéve mindenhonnan